manifests/v/Vencord/Vencord/PTB/1.2.10/Vencord.Vencord.PTB.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Vencord.Vencord.PTB
+PackageVersion: 1.2.10
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord.PTB
+Installers:
+- Architecture: x64

```

```suggestion
- Architecture: x86
```
I'm not sure, but I believe the architecture has to be `x86` for the Discord call to work correctly.